src/main/groovy/life/qbic/portal/sampletracking/app/samples/list/ModifySampleListInput.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.portal.sampletracking.app.samples.list
+
+import life.qbic.datamodel.samples.Sample
+
+interface ModifySampleListInput {
+    void selectSamples(Sample... samples)
+
+    void deselectSamples(Sample... samples)
+
+    void clearSelection()
+
+    void injectSampleListOutput(ModifySampleListOutput sampleListOutput)
```

Comment:
```suggestion
```

Reply:
I cannot commit this since the method is used in the constructor and needs to be refactored which cannot be done by this suggestion.